magento/magento2
# FIx text and images are displayed cropped if folder has long name && Filters isn`t applied on the "Category"

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1758: Text and images are displayed cropped if folder has long name
2. Fixes magento/adobe-stock-integration#1763: Filters isn`t applied on the "Category" 

### Manual testing scenarios (*)
1. Go to Content -> Media Gallery
2. Create a folder with long name (for example: `TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEESSSSSSSSSSSSSSSSSSSSSSSTTTTTTTTTTTTTT`)
3. Select it

"We couldn't find any records." text should be displayed without cropping it there is no any image. If there is any image, it should also be displayed without cropping.
1. Go to Content -> Media Gallery
2. Select uploaded image
3. Click on three dots -> View Details 
4. Click on the link 'Categories..." in 'Used in' attribute
5. Delete the `?filters[asset_id]=[1]` part of request from address line -> Press `Enter` key
6. Pay attention to the 'Active filters:'
7. Click on the "Clear all" link
8. Waite a few seconds and reload the page
9. Click on the "Filters" button
10. Enter any word in the "ID" field -> Apply Filters
11. Try to apply any filter

Filter should be applied 

### Questions or comments


### Contribution checklist (*)
